### PR TITLE
FIX: InvalidOperationException thrown when selecting a preset created from an InputActionImporter

### PR DESCRIPTION
**Pull request by andrew-oc**
[case 1199544](https://issuetracker.unity3d.com/issues/input-system-properties-are-not-visible-and-invalidoperationexception-is-thrown-on-selecting-inputactionimporter-preset-asset) ([Fogbugz](https://fogbugz.unity3d.com/f/cases/1199544/))

### Description

The InputActionImporterEditor expected the assetTarget field to be set i.e. that the currently selected asset was an InputActionAsset, but when selecting a preset asset, this wasn't the case because presets don't save asset references. 

### Changes made

Added a warning message to the InputActionImporterEditor when the selected object is not an InputActionAsset and disabled the edit button in this case.

### Checklist

Before review:

- [x] Changelog entry added.
    - Explains the change in `Changed`, `Fixed`, `Added` sections.
    - For API change contains an example snippet and/or migration example.
    - FogBugz ticket attached, example `([case %number%](https://issuetracker.unity3d.com/issues/...))`.
    - FogBugz is marked as "Resolved" with *next* release version correctly set.
- [ ] Tests added/changed, if applicable.
    - Functional tests `Area_CanDoX`, `Area_CanDoX_EvenIfYIsTheCase`, `Area_WhenIDoX_AndYHappens_ThisIsTheResult`.
    - Performance tests.
    - Integration tests.
- [x] Docs for new/changed API's.
    - Xmldoc cross references are set correctly.
    - Added explanation how the API works.
    - Usage code examples added.
    - The manual is updated, if needed.

During merge:

- [x] Commit message for squash-merge is prefixed with one of the list:
    - `NEW: ___`.
    - `FIX: ___`.
    - `DOCS: ___`.
    - `CHANGE: ___`.
    - `RELEASE: 1.1.0-preview.3`.

### CODE REVIEW

File: Packages/com.unity.inputsystem/CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ however, it has to be formatted properly to pass verification tests.
 
 - Fixed pairing devices to existing `InputUser`s potentially corrupting list of paired devices from other `InputUser`s ([case 1327628](https://issuetracker.unity3d.com/issues/input-system-devices-are-reassigned-to-the-wrong-users-after-adding-a-new-device)).
 - Fixed StackOverflowException caused by calling InputSystem.Update from inside an input action callback ([case 1316000](https://issuetracker.unity3d.com/issues/crash-when-adding-inputsystem-dot-update-to-inputsystem-command-handler-to-force-processing-an-event-and-sending-input)).
+- Fixed InvalidOperationException when opening a preset created from an InputActionImporter ([case 1199544](https://issuetracker.unity3d.com/issues/input-system-properties-are-not-visible-and-invalidoperationexception-is-thrown-on-selecting-inputactionimporter-preset-asset)).
```
Comment (Rene-Damm) [inline Packages/com.unity.inputsystem/CHANGELOG.md]: Minor... `InputActionImporter` is an internal class. "preset create from an .inputactions asset" probably clearer.

File: Packages/com.unity.inputsystem/InputSystem/Editor/AssetEditor/InputActionEditorWindow.cs [hunk 2]
```diff
@@ -235,6 +247,9 @@ private void OnDestroy()
         // Set asset would usually only be called when the window is open
         private void SetAsset(InputActionAsset asset)
         {
+            if (asset == null)
+                return;
```
Comment (Rene-Damm) [inline Packages/com.unity.inputsystem/InputSystem/Editor/AssetEditor/InputActionEditorWindow.cs:259]: This case looks malformed. .inputactions editor windows with no associated assets should get nuked, it seems. What's the flow where we run into this if?

Comment (andrew-oc) [reply Packages/com.unity.inputsystem/InputSystem/Editor/AssetEditor/InputActionEditorWindow.cs:259]: The OpenEditor method, which calls SetAsset, is public and takes the asset as an argument, so it seems better to check it for null. I didn't see that the editor window gets blown away if there's no asset. I was definitely able to open one with this being null.

**Review by ekcoh — APPROVED** [review]
Changes looks good to me.

**Pull request merged**